docker/Dockerfile.dev.ui

```diff
@@ -10,7 +10,7 @@ WORKDIR /app
 
 # Copy package.json and package-lock.json
 # to the /app working directory
-COPY keep-ui/package*.json /app
+COPY keep-ui/package*.json /app/
```

Comment:
I was receiving a build error that I needed a trailing slash for this `COPY`. I can remove this from the PR if it is only affecting my docker env